e2e-tests/scaling/run

```diff
@@ -52,5 +52,25 @@ kubectl_bin patch psmdb ${cluster%%-rs0} \
 desc 'check if Pod deleted'
 wait_for_delete pod/$cluster-3
 wait_for_delete pod/$cluster-4
+wait_for_running $cluster 3
+
+desc 'scale up from 3 to 1'
+kubectl_bin patch psmdb ${cluster%%-rs0} \
```

Comment:
it looks like that we have the problem only if cluster expose. so we should check this case too. maybe we can put it in expose test.